audit/audit_syslog_storage_helper.cc

```diff
@@ -54,6 +54,18 @@ static auto syslog_address_helper(const db::config& cfg)
         : unix_domain_addr(_PATH_LOG);
 }
 
+static auto json_escape(const sstring& str) {
```

Comment:
Nitpick for future consideration (I don't consider it critical now):
Please avoid "auto" in declaring functions as much as possible. It's useful for templates with very complicated return types, but for ordinary functions, it just obscures the fact that this function always returns an `std::string`, a fact that readers of this function will want to know.

Another nitpick: you could have taken an `std::string_view` instead of a `const sstring&`, and it would have allowed taking both sstring and std::string as parameter.